ios/RNGoogleMobileAds/RNGoogleMobileAdsBannerComponent.m

```diff
@@ -78,6 +83,14 @@ - (void)setSizes:(NSArray *)sizes {
   _propsChanged = true;
 }
 
+- (void)setMaxAdHeight:(CGFloat)maxAdHeight {
+  _maxAdHeight = maxAdHeight;
+  if (_sizeStrings != nil) {
```

Comment:
From my testing in the example app, this condition never becomes truthy. So do we really need this extra sizeStrings?

Reply:
Yes, I couldn't reproduce this on iOS or Android, but I was missing the same check on Android initially and running this with ~700k Android users in production, I was getting 10s of crashes a day.

My plan is to refactor this to pass an object with the size related props across to native in a single prop, while having separate props exposed on the JS side.